### PR TITLE
Assertion and logging for Parser and addToWatchlist

### LINKED ISSUE #88: Implement Assertions and Logging for Parser class and addToWatchlist feature

### PR DESCRIPTION
**Pull request by michaeldinata**
Added:
1. Some logger and assertions into Parser and AddToWatchlistCommand class
2. Refactored AddToWatchlistCommand class

closes #88

### CODE REVIEW

File: src/main/java/seedu/duke/command/AddToWatchlistCommand.java [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
 import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class AddToWatchlistCommand extends Command {
+    private static final String ADD_OPTION = "-a";
+    
+    private String option;
+    private String animeName = "";
+    private static Logger logger = Logger.getLogger("AddToWatchlistLogger");
```
Comment (ChanJianHao) [inline src/main/java/seedu/duke/command/AddToWatchlistCommand.java]: Perhaps we could standardize our logger to be something along the lines of private static `Logger LOGGER = Logger.getLogger("<CLASS>.class.getName()");`

**Review by ChanJianHao — APPROVED** [review]
LGTM, thanks for refactoring! 👍🏻

**Review by EyoWeiChin — APPROVED** [review]
LGTM, Thanks!

File: src/main/java/seedu/duke/command/AddToWatchlistCommand.java (hunk 1 above)
Comment (michaeldinata) [reply src/main/java/seedu/duke/command/AddToWatchlistCommand.java]: The logger has been updated for all the classes that I could find it in. Thanks for pointing it out!

**Pull request merged**